breathe/parser/compoundsuper.py

```diff
@@ -2436,14 +2442,22 @@ def hasContent_(self):
             return True
         else:
             return False
-    def build(self, node_):
+    def build(self, node_: minidom.Element):
         attrs = node_.attributes
         self.buildAttributes(attrs)
         for child_ in node_.childNodes:
             nodeName_ = child_.nodeName.split(':')[-1]
             self.buildChildren(child_, nodeName_)
-    def buildAttributes(self, attrs):
-        pass
+    def buildAttributes(self, attrs: minidom.NamedNodeMap):
+        if "filename" in attrs.keys():
+            # extract the domain for this programlisting tag.
+            file_ext = list(os.path.splitext(attrs["filename"].value))
+            try:
+                lexer_cls = pygments.lexers.get_lexer_for_filename(attrs["filename"].value)
+                file_ext[1] = lexer_cls.name.lower()
```

Comment:
found from testing that a \*.txt returns `text only` as the `lexer_cls.name`. Use `lexer_cls.aliases[0]` instead
```suggestion
                file_ext[1] = lexer_cls.aliases[0]
```